packages/dev-server-esbuild/package.json

```diff
@@ -52,7 +52,7 @@
   "dependencies": {
     "@mdn/browser-compat-data": "^4.0.0",
     "@web/dev-server-core": "^0.3.17",
-    "esbuild": "^0.12.21",
+    "esbuild": "^0.12|^0.13|^0.14",
```

Comment:
Is `0.14` “latest”? Would we benefit from `>0.12` to simplify in the future?

Reply:
yes, `0.14` is latest. having the `|` in there with 12, 13 and 14 is more restrictive than `>0.12`. I can live with both.